apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetDAO.java

```diff
@@ -76,4 +77,18 @@ List<Dataset> find(@Bind("limit") int limit,
     @SqlQuery("SELECT * FROM datasets WHERE workspace_id = :workspace_id AND name = :name")
     Optional<Dataset> findByName(@Bind("workspace_id") String workspaceId, @Bind("name") String name);
 
+    default int[] recordExperiments(Handle handle, String workspaceId,
```

Comment:
Not a blocker: you're mixing the fluent and declarative interfaces of JDBI. I believe you can implement batches with the following annotation: https://jdbi.org/#_sqlbatch

Reply:
I tried, but it seems this only works for inserts.  

I can try again later, but the bind of the collection didn't work
https://jdbi.org/#_prepared_batches